pyquidax/wrappers/beneficiary.py

```diff
@@ -5,16 +5,51 @@
 
 
 class Beneficiary(BaseAPIWrapper):
-    def __init__(self, secret_key: Optional[str] = None):
-        super().__init__(secret_key)
+    """A wrapper for interacting with user accounts on Quidax"""
 
     def all(self, currency: Currency, user_id: str = "me"):
+        """Fetch all beneficiaries for the authenticated user or a sub account
+
+        Args:
+            currency: Currency Allowed valued: btc, ltc, xrp, dash, trx, doge.
+
```

Comment:
You can change the to their currency enums equivalent e.g. `Currency.BTC` check the `Currency` enum